addons/website_sale/data/product_snippet_template_data.xml

```diff
@@ -80,8 +88,12 @@
                     <a class="o_carousel_product_img_link" t-att-href="record.website_url">
                         <img class="card-img-top p-3" loading="lazy" t-att-src="data['image_512']" t-att-alt="record.display_name"/>
                     </a>
-                    <div class="card-text text-center" t-esc="data['price']"
-                        t-options="{'widget': 'monetary', 'display_currency': website.currency_id}"/>
+                    <div class="card-text text-center pb-1">
+                        <span t-esc="data['price']"
+                            t-options="{'widget': 'monetary', 'display_currency': website.currency_id}"/>
```

Comment:
```suggestion
                        <span t-esc="data['price']"
                              t-options="{'widget': 'monetary', 'display_currency': website.currency_id}"/>
```
```suggestion
                              t-options="{'widget': 'monetary', 'display_currency': website.currency_id}"/>
```
Make attribute on new line start at the same level as the previous one otherwise it is ugly :innocent: 
There is a new spec to never split on multiple line a line whatever the length of that line.. I don't agree with it and I don't think it is official so.. I'd say chose one or the other :)

Reply:
Just so you know, AFAIK, there is a :fire: debate about this on another PR to introduce a guideline related to that.
There is so many possibilities (one-line, multi-line, indent 4 space, indent tag length etc) and it seems it will just be left to the developer preferences so anything will work as long as it is consistent I guess :) 

What I suggested here was just how I prefer it but actually you can write it as you prefer :+1: so you can ignore my previous message if you did not agree :)

In case it helps, this is what I prefer:
- Never one-line if the line is too long, multi-line it (especially helps people who doesn't soft wrap in their IDE, it avoids having to scroll to the right too much)
- Indent the same level as the tag, especially if it is 2 lines tag
- Don't have one attribute per line if the attribute is short..